couch_tap.gemspec

```
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.add_dependency "byebug", "11.0.1"
```

I think we do not require a particular version of byebug, so probably something >= 11.0 would do. At least greater than 11.0 but lower than 12, what do you think?
```suggestion
  s.add_dependency "byebug", "~> 11.0"
```

---

Agree. Done in 72ce7a9.